docs/docs/0.3.2/documentation/README.md

```diff
@@ -1841,8 +1847,40 @@ When `kuma-cp` starts up, by default it listens on a few ports:
 * `5682`: the HTTP server that provides the Envoy bootstrap configuration when the data-plane starts up.
 * `5683`: the HTTP server that exposes Kuma UI.
 
-## Quickstart
+### kuma-dp ports
+
+When we start a data-plane via `kuma-dp` we expect all the inbound and outbound service traffic to go through it. The inbound and outbound ports are defined in the [dataplane specification](#dataplane-specification) when running in universal mode, while on Kubernetes the service traffic always runs on port `15001`.
+
+In addition to the service traffic ports, the data-plane automatically also opens the `envoy` [administration interface](https://www.envoyproxy.io/docs/envoy/latest/operations/admin) service on the following ports:
+
+* On Kubernetes, by default on port `9901`.
+* On Universal, by default on the first available port greater or equal than `30001`.
+
+The Envoy administration interface can also be manually configured to listen on any arbitraty port by specifying the `--admin-port` argument when running `kuma-dp`.
+
+### Service Discovery
+
+Here we are going to be exploring the communication between `kuma-dp` and `kuma-cp`, and the communication between multiple `kuma-dp` to handle our service traffic.
+
+Every time a data-plane (served by `kuma-dp`) registers itself to Kuma, it initiates a gRPC streaming connection to Kuma (served by `kuma-cp`) in order to retrieve the latest configuration and share data to the control-plane. 
```

Comment:
> and share data to the control-plane

is definitely ambiguous.

What data does it share with the Control Plane ? Intercepted traffic ?